csrc/multidevice/communicator.cpp

```diff
@@ -343,8 +357,6 @@ void Communicator::cleanup() {
   }
 #endif
   backends_.clear();
-
-  is_available_ = false;
```

Comment:
I moved it up to the other use of is_available_. This way, it's easier to remember to reset is_available_.